icons/rockylinux/rockylinux-original-wordmark.svg

```diff
@@ -0,0 +1 @@
781 L 110.679688 61.371094 L 111.925781 61.371094 L 111.925781 69.777344 L 110.679688 69.777344 L 110.679688 68.804688 C 110.355469 69.183594 109.964844 69.46875 109.519531 69.660156 C 109.082031 69.851562 108.589844 69.945312 108.039062 69.945312 C 107.074219 69.945312 106.289062 69.644531 105.679688 69.039062 C 105.074219 68.433594 104.773438 67.652344 104.773438 66.683594 L 104.773438 61.363281 L 106 61.363281 Z M 113.066406 69.78125 L 116.378906 65.441406 L 113.242188 61.359375 L 114.722656 61.359375 L 117.101562 64.519531 L 119.480469 61.359375 L 120.902344 61.359375 L 117.800781 65.417969 L 121.125 69.769531 L 119.644531 69.769531 L 117.078125 66.339844 L 114.476562 69.769531 L 113.066406 69.769531 Z M 113.066406 69.78125 "/><path  fill="#000000" d="M 122.183594 60.933594 L 122.183594 59.113281 L 121.578125 59.113281 L 121.578125 58.761719 L 122.183594 58.761719 L 122.183594 58.050781 L 122.589844 57.945312 L 122.589844 58.761719 L 123.4375 58.761719 L 123.4375 59.113281 L 122.589844 59.113281 L 122.589844 60.828125 C 122.589844 60.976562 122.625 61.085938 122.691406 61.152344 C 122.757812 61.214844 122.871094 61.246094 123.023438 61.246094 C 123.101562 61.246094 123.167969 61.242188 123.230469 61.230469 C 123.292969 61.21875 123.359375 61.199219 123.429688 61.175781 L 123.429688 61.539062 C 123.359375 61.5625 123.28125 61.582031 123.191406 61.59375 C 123.105469 61.605469 123.019531 61.609375 122.9375 61.609375 C 122.695312 61.609375 122.507812 61.554688 122.378906 61.4375 C 122.25 61.320312 122.183594 61.152344 122.183594 60.933594 Z M 123.886719 61.558594 L 123.886719 58.761719 L 124.300781 58.761719 L 124.300781 59.070312 C 124.402344 58.949219 124.523438 58.859375 124.664062 58.800781 C 124.800781 58.738281 124.953125 58.707031 125.121094 58.707031 C 125.316406 58.707031 125.492188 58.753906 125.644531 58.84375 C 125.800781 58.933594 125.921875 59.058594 126.007812 59.214844 C 126.121094 59.042969 126.261719 58.917969 126.421875 58.835938 C 126.585938 58.75 126.769531 58.707031 126.96875 58.707031 C 127.273438 58.707031 127.519531 58.808594 127.707031 59.007812 C 127.902344 59.210938 128 59.472656 128 59.792969 L 128 61.566406 L 127.589844 61.566406 L 127.589844 59.875 C 127.589844 59.628906 127.523438 59.433594 127.394531 59.285156 C 127.265625 59.140625 127.085938 59.070312 126.863281 59.070312 C 126.710938 59.070312 126.570312 59.109375 126.441406 59.1875 C 126.316406 59.265625 126.207031 59.378906 126.117188 59.527344 C 126.125 59.570312 126.132812 59.613281 126.136719 59.65625 C 126.144531 59.695312 126.148438 59.742188 126.148438 59.789062 L 126.148438 61.5625 L 125.738281 61.5625 L 125.738281 59.871094 C 125.738281 59.625 125.671875 59.429688 125.542969 59.285156 C 125.410156 59.136719 125.234375 59.066406 125.015625 59.066406 C 124.867188 59.066406 124.730469 59.101562 124.605469 59.171875 C 124.488281 59.238281 124.382812 59.339844 124.300781 59.472656 L 124.300781 61.5625 L 123.886719 61.5625 Z M 123.886719 61.558594 "/></svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128" ><path  fill="#10b982" d="M 21.769531 67.695312 C 22.171875 66.542969 22.390625 65.292969 22.390625 64 C 22.390625 57.820312 17.378906 52.804688 11.195312 52.804688 C 5.015625 52.804688 0 57.820312 0 64 C 0 67.054688 1.222656 69.832031 3.21875 71.847656 L 14.578125 60.488281 L 21.773438 67.683594 Z M 19.726562 71.253906 L 14.574219 66.097656 L 6.503906 74.167969 C 7.925781 74.828125 9.511719 75.195312 11.191406 75.195312 C 14.609375 75.195312 17.664062 73.667969 19.714844 71.253906 Z M 19.726562 71.253906 "/><path fill="#000000" d="M 29.234375 69.796875 L 29.234375 58.015625 L 34.972656 58.015625 C 35.566406 58.015625 36.105469 58.105469 36.605469 58.285156 C 37.109375 58.464844 37.539062 58.71875 37.898438 59.042969 C 38.257812 59.355469 38.535156 59.734375 38.738281 60.183594 C 38.941406 60.628906 39.039062 61.121094 39.039062 61.664062 C 39.039062 62.402344 38.828125 63.0625 38.402344 63.648438 C 37.988281 64.230469 37.433594 64.660156 36.734375 64.941406 L 39.335938 69.78125 L 36.617188 69.78125 L 34.320312 65.28125 L 31.660156 65.28125 L 31.660156 69.78125 L 29.222656 69.78125 Z M 34.761719 60.140625 L 31.671875 60.140625 L 31.671875 63.300781 L 34.761719 63.300781 C 35.308594 63.300781 35.746094 63.15625 36.066406 62.863281 C 36.402344 62.574219 36.570312 62.191406 36.570312 61.722656 C 36.570312 61.253906 36.402344 60.871094 36.066406 60.582031 C 35.742188 60.289062 35.304688 60.144531 34.761719 60.144531 Z M 39.566406 65.433594 C 39.566406 64.796875 39.683594 64.199219 39.917969 63.648438 C 40.164062 63.089844 40.496094 62.609375 40.910156 62.203125 C 41.335938 61.789062 41.832031 61.464844 42.402344 61.230469 C 42.972656 60.996094 43.582031 60.878906 44.234375 60.878906 C 44.886719 60.878906 45.492188 60.996094 46.0625 61.230469 C 46.636719 61.464844 47.128906 61.789062 47.546875 62.203125 C 47.972656 62.609375 48.300781 63.089844 48.535156 63.648438 C 48.78125 64.199219 48.90625 64.792969 48.90625 65.433594 C 48.90625 66.078125 48.78125 66.671875 48.535156 67.230469 C 48.300781 67.777344 47.972656 68.253906 47.546875 68.652344 C 47.132812 69.058594 46.636719 69.375 46.0625 69.609375 C 45.492188 69.847656 44.886719 69.964844 44.234375 69.964844 C 43.582031 69.964844 42.972656 69.84375 42.402344 69.609375 C 41.832031 69.375 41.332031 69.058594 40.910156 68.652344 C 40.496094 68.25 40.164062 67.773438 39.917969 67.230469 C 39.683594 66.671875 39.566406 66.070312 39.566406 65.433594 Z M 44.230469 67.964844 C 44.902344 67.964844 45.480469 67.71875 45.945312 67.226562 C 46.410156 66.734375 46.652344 66.132812 46.652344 65.429688 C 46.652344 64.714844 46.414062 64.113281 45.945312 63.609375 C 45.476562 63.109375 44.902344 62.871094 44.230469 62.871094 C 43.558594 62.871094 42.984375 63.117188 42.515625 63.609375 C 42.050781 64.105469 41.8125 64.707031 41.8125 65.429688 C 41.8125 66.136719 42.046875 66.734375 42.515625 67.226562 C 42.988281 67.714844 43.558594 67.964844 44.230469 67.964844 Z M 54.65625 67.949219 C 55.050781 67.949219 55.414062 67.871094 55.75 67.710938 C 56.097656 67.554688 56.425781 67.308594 56.738281 66.972656 L 58.117188 68.394531 C 57.667969 68.886719 57.128906 69.277344 56.507812 69.554688 C 55.878906 69.824219 55.234375 69.960938 54.558594 69.960938 C 53.921875 69.960938 53.3125 69.839844 52.738281 69.605469 C 52.179688 69.371094 51.691406 69.054688 51.28125 68.648438 C 50.878906 68.246094 50.558594 67.769531 50.324219 67.226562 C 50.089844 66.667969 49.972656 66.066406 49.972656 65.429688 C 49.972656 64.792969 50.089844 64.195312 50.324219 63.644531 C 50.558594 63.085938 50.878906 62.605469 51.28125 62.199219 C 51.695312 61.785156 52.183594 61.460938 52.738281 61.226562 C 53.3125 60.992188 53.917969 60.875 54.558594 60.875 C 55.253906 60.875 55.921875 61.015625 56.554688 61.292969 C 57.191406 61.5625 57.730469 61.945312 58.1875 62.4375 L 56.773438 63.929688 C 56.460938 63.582031 56.125 63.320312 55.765625 63.140625 C 55.410156 62.960938 55.023438 62.871094 54.609375 62.871094 C 53.949219 62.871094 53.382812 63.117188 52.929688 63.609375 C 52.480469 64.101562 52.257812 64.707031 52.257812 65.429688 C 52.257812 66.152344 52.488281 66.746094 52.945312 67.226562 C 53.414062 67.707031 53.988281 67.949219 54.660156 67.949219 Z M 59.308594 69.792969 L 59.308594 58.011719 L 61.609375 57.507812 L 61.609375 64.714844 L 65.386719 61.019531 L 67.988281 61.019531 L 63.894531 65.03125 L 68.242188 69.765625 L 65.304688 69.765625 L 61.609375 65.765625 L 61.609375 69.765625 L 59.308594 69.765625 Z M 71.441406 70.195312 L 71.574219 69.875 L 68.167969 61.046875 L 70.699219 61.046875 L 72.867188 67.007812 L 75.351562 61.046875 L 77.835938 61.046875 L 73.800781 70.480469 C 73.351562 71.535156 72.832031 72.289062 72.238281 72.730469 C 71.644531 73.175781 70.875 73.402344 69.917969 73.402344 C 69.714844 73.402344 69.519531 73.394531 69.328125 73.371094 C 69.148438 73.359375 69.003906 73.335938 68.894531 73.304688 L 68.894531 71.320312 C 69.003906 71.34375 69.125 71.359375 69.246094 71.371094 C 69.367188 71.382812 69.519531 71.386719 69.699219 71.386719 C 70.113281 71.386719 70.460938 71.289062 70.742188 71.085938 C 71.03125 70.882812 71.261719 70.585938 71.429688 70.195312 Z M 82.414062 69.792969 L 82.414062 58.011719 L 83.707031 58.011719 L 83.707031 68.613281 L 90.707031 68.613281 L 90.707031 69.773438 L 82.414062 69.773438 Z M 92.933594 59.761719 C 92.710938 59.761719 92.511719 59.675781 92.34375 59.507812 C 92.175781 59.339844 92.09375 59.144531 92.09375 58.921875 C 92.09375 58.6875 92.175781 58.488281 92.34375 58.332031 C 92.511719 58.164062 92.710938 58.082031 92.933594 58.082031 C 93.167969 58.082031 93.363281 58.164062 93.519531 58.332031 C 93.6875 58.488281 93.773438 58.6875 93.773438 58.921875 C 93.773438 59.144531 93.6875 59.339844 93.519531 59.507812 C 93.363281 59.675781 93.167969 59.761719 92.933594 59.761719 Z M 93.554688 61.371094 L 93.554688 69.78125 L 92.308594 69.78125 L 92.308594 61.371094 Z M 95.675781 69.78125 L 95.675781 61.371094 L 96.925781 61.371094 L 96.925781 62.363281 C 97.25 61.980469 97.636719 61.695312 98.085938 61.507812 C 98.53125 61.304688 99.03125 61.203125 99.578125 61.203125 C 100.527344 61.203125 101.304688 61.507812 101.910156 62.113281 C 102.515625 62.714844 102.816406 63.5 102.816406 64.46875 L 102.816406 69.785156 L 101.59375 69.785156 L 101.59375 64.710938 C 101.59375 63.972656 101.378906 63.382812 100.953125 62.949219 C 100.527344 62.519531 99.957031 62.296875 99.242188 62.296875 C 98.738281 62.296875 98.28125 62.40625 97.875 62.632812 C 97.484375 62.855469 97.164062 63.175781 96.917969 63.589844 L 96.917969 69.78125 L 95.671875 69.78125 Z M 106.011719 61.371094 L 106.011719 66.445312 C 106.011719 67.183594 106.222656 67.773438 106.648438 68.203125 C 107.074219 68.636719 107.644531 68.859375 108.363281 68.859375 C 108.867188 68.859375 109.3125 68.75 109.703125 68.523438 C 110.109375 68.289062 110.433594 67.964844 110.679688 67.550781 L 110.679688 61.371094 L 111.925781 61.371094 L 111.925781 69.777344 L 110.679688 69.777344 L 110.679688 68.804688 C 110.355469 69.183594 109.964844 69.46875 109.519531 69.660156 C 109.082031 69.851562 108.589844 69.945312 108.039062 69.945312 C 107.074219 69.945312 106.289062 69.644531 105.679688 69.039062 C 105.074219 68.433594 104.773438 67.652344 104.773438 66.683594 L 104.773438 61.363281 L 106 61.363281 Z M 113.066406 69.78125 L 116.378906 65.441406 L 113.242188 61.359375 L 114.722656 61.359375 L 117.101562 64.519531 L 119.480469 61.359375 L 120.902344 61.359375 L 117.800781 65.417969 L 121.125 69.769531 L 119.644531 69.769531 L 117.078125 66.339844 L 114.476562 69.769531 L 113.066406 69.769531 Z M 113.066406 69.78125 "/><path  fill="#000000" d="M 122.183594 60.933594 L 122.183594 59.113281 L 121.578125 59.113281 L 121.578125 58.761719 L 122.183594 58.761719 L 122.183594 58.050781 L 122.589844 57.945312 L 122.589844 58.761719 L 123.4375 58.761719 L 123.4375 59.113281 L 122.589844 59.113281 L 122.589844 60.828125 C 122.589844 60.976562 122.625 61.085938 122.691406 61.152344 C 122.757812 61.214844 122.871094 61.246094 123.023438 61.246094 C 123.101562 61.246094 123.167969 61.242188 123.230469 61.230469 C 123.292969 61.21875 123.359375 61.199219 123.429688 61.175781 L 123.429688 61.539062 C 123.359375 61.5625 123.28125 61.582031 123.191406 61.59375 C 123.105469 61.605469 123.019531 61.609375 122.9375 61.609375 C 122.695312 61.609375 122.507812 61.554688 122.378906 61.4375 C 122.25 61.320312 122.183594 61.152344 122.183594 60.933594 Z M 123.886719 61.558594 L 123.886719 58.761719 L 124.300781 58.761719 L 124.300781 59.070312 C 124.402344 58.949219 124.523438 58.859375 124.664062 58.800781 C 124.800781 58.738281 124.953125 58.707031 125.121094 58.707031 C 125.316406 58.707031 125.492188 58.753906 125.644531 58.84375 C 125.800781 58.933594 125.921875 59.058594 126.007812 59.214844 C 126.121094 59.042969 126.261719 58.917969 126.421875 58.835938 C 126.585938 58.75 126.769531 58.707031 126.96875 58.707031 C 127.273438 58.707031 127.519531 58.808594 127.707031 59.007812 C 127.902344 59.210938 128 59.472656 128 59.792969 L 128 61.566406 L 127.589844 61.566406 L 127.589844 59.875 C 127.589844 59.628906 127.523438 59.433594 127.394531 59.285156 C 127.265625 59.140625 127.085938 59.070312 126.863281 59.070312 C 126.710938 59.070312 126.570312 59.109375 126.441406 59.1875 C 126.316406 59.265625 126.207031 59.378906 126.117188 59.527344 C 126.125 59.570312 126.132812 59.613281 126.136719 59.65625 C 126.144531 59.695312 126.148438 59.742188 126.148438 59.789062 L 126.148438 61.5625 L 125.738281 61.5625 L 125.738281 59.871094 C 125.738281 59.625 125.671875 59.429688 125.542969 59.285156 C 125.410156 59.136719 125.234375 59.066406 125.015625 59.066406 C 124.867188 59.066406 124.730469 59.101562 124.605469 59.171875 C 124.488281 59.238281 124.382812 59.339844 124.300781 59.472656 L 124.300781 61.5625 L 123.886719 61.5625 Z M 123.886719 61.558594 "/></svg>

```
missing new line

Reply:
Thanks for the heads up. I've added the new line for each SVG. Will do the same for the others PRs.